docs/strategies/circuit-breaker.md

```diff
@@ -37,6 +37,19 @@ new ResiliencePipelineBuilder().AddCircuitBreaker(new CircuitBreakerStrategyOpti
     ShouldHandle = new PredicateBuilder().Handle<SomeExceptionType>()
 });
 
+// Adds a circuit breaker with dynamic break duration:
+//
+// Same circuit breaking conditions as above, but with a dynamic break duration based on the failure count.
+// The duration is calculated as: minimum of (20 + 2^failureCount) seconds and capped at 400 seconds.
+new ResiliencePipelineBuilder().AddCircuitBreaker(new CircuitBreakerStrategyOptions
+{
+    FailureRatio = 0.5,
+    SamplingDuration = TimeSpan.FromSeconds(10),
+    MinimumThroughput = 8,
+    ShouldHandle = new PredicateBuilder().Handle<SomeExceptionType>(),
+    BreakDurationGenerator = (args) => TimeSpan.FromSeconds(Math.Min(20 + Math.Pow(2, args.FailureCount), 400))
```

Comment:
Please consider to use `static anonymous function`

```suggestion
BreakDurationGenerator = static args => TimeSpan.FromSeconds(Math.Min(20 + Math.Pow(2, args.FailureCount), 400))
```

Reply:
Also please list the `BreakDurationGenerator` under the *Defaults* section (right after the `BreakDuration`.

Reply:
@atawLee Please don't forget to extend the `Defaults` table with the `BreakDurationGenerator` (right after the BreakDuration).